tests/helpers/strapi.js

```diff
@@ -20,4 +20,28 @@ async function setupStrapi() {
   }
   return instance;
 }
-module.exports = { setupStrapi };
+
+/**
+ * Grants database `permissions` table that role can access an endpoint/controllers
+ *
+ * @param {int} roleID, 1 Autentihected, 2 Public, etc
+ * @param {string} value, in form or dot string eg `"permissions.users-permissions.controllers.auth.changepassword"`
+ * @param {boolean} enabled, default true
+ * @param {string} policy, default ''
+ */
+ const grantPriviledge = async (
+  roleID = 1,
+  value,
+  enabled = true,
+  policy = ""
+) => {
+  const updateObj = value
+    .split(".")
+    .reduceRight((obj, next) => ({ [next]: obj }), { enabled, policy });
+
```

Comment:
Mikä on tän grantPriviledgen idea ja mitä tää updateObjectin parsiminen käytännössä tekee?

Reply:
I updated the docs to be a bit more clear, they should answer your question.

The `updateObj` variable is just an object that Strapi's utility `updateRole` understands.

With a call: 
```javascript
grantPriviledge(2, 'permissions.application.controllers.suggestion.new')
```

the updateObj is: 
```javascript
{
  permissions: {
    application: {
      controllers: { 
        suggestion: { 
          new: { 
            enabled: true, 
            policy: "" 
          } 
        } 
      },
    },
  },
};
```